python/npcomp/compiler/pytorch/backend/refjit.py

```diff
@@ -68,9 +84,9 @@ def compile(self, imported_module: Module):
         imported_module, refjit_backend.get_runtime_libs())
     return jit_module
 
-  def load(self, jit_module):
+  def load(self, jit_module) -> TorchJitModuleInvoker:
     """Loads a compiled artifact into the runtime.
 
-    Since this is a JIT instead of an AOT compiler,
+    Since this is a JIT instead of an AOT compiler, TODO: finish this sentence.
```

Comment:
Please complete this sentence. I would but I can only think of really obvious things to say, like "...it compiles it just in time".